system/lib/wasmfs/file.h

```diff
@@ -69,7 +99,15 @@ class File : public std::enable_shared_from_this<File> {
   std::mutex mutex;
 
 private:
-  // TODO: Add other File properties later.
+  size_t usedBytes;
+
+  uint32_t uid;  // User ID of the owner
+  uint32_t gid;  // Group ID of the owning group
```

Comment:
Emscripten doesn't have multiple users in the POSIX sense, so these can probably also be hardcoded in `stat`. Unless the current file system supports these fields?